lang/russian.json

```diff
@@ -1,146 +1,146 @@
 {
     "__lang__": "ru",
-    "aboutAutomad": "О сайте Automad",
+    "aboutAutomad": "Об Automad",
     "add": "Добавить",
     "addImage": "Добавить изображение",
-    "addPage": "Добавить страницу",
-    "addShadow": "Добавить тень",
+    "addPage": "Добавить Страницу",
+    "addShadow": "Добавить Тень",
     "addWatermark": "Добавить водяной знак",
     "addWatermarkAsText": "Добавить как текст",
-    "addWatermarkTitle": "Выбери тип водяного знака",
-    "addedSuccess": "Успешно добавлено",
-    "adjustTab": "Настрой",
-    "alignCenter": "Выравнивание по центру",
-    "alignLeft": "Выравнивание по левому краю",
-    "alignRight": "Выравнивание по правому краю",
+    "addWatermarkTitle": "Выберите тип водяного знака",
+    "addedSuccess": "Успешно добавлен(о)",
+    "adjustTab": "Настроить",
+    "alignCenter": "Выровнять по центру",
+    "alignLeft": "Выровнять по левому краю",
+    "alignRight": "Выровнять по правому краю",
     "alreadyExists": "уже существует!",
     "annotateTab": "Рисовать",
     "areNotImages": "не являются изображениями",
-    "arrowTool": "Стрелка",
+    "arrowTool": "Стрелка (указатеь)",
     "back": "Назад",
     "backgroundColor": "Цвет фона",
-    "backgroundImage": "Фоновое изображение",
-    "blockquote": "Blockquote",
-    "blur": "Размытие",
+    "backgroundImage": "Изображение фона",
+    "blockquote": "Блок-цитата",
```

Comment:
Blockquote is a fixed term and HTML element. That is not translated on purpose. Please keep it as is.